src/enums/routeNames.ts

```diff
@@ -11,5 +11,7 @@ export enum RouteNames {
     CREATE_SHARE_STRUCTURE = 'create-share-structure',
     DEFINE_COMPANY = 'define-company',
     INCORPORATION_AGREEMENT = 'incorporation-agreement',
```

Comment:
Don't need these other ones?

Reply:
At this time, i don't believe so. From what i see in the wireframes, we are a single view for an entire 'Correction' or 'Alteration' and so on. So i believe it will be up to us what components we populate in each respective views, dependant on the type of filing we are correcting, altering etc.

Reply:
Will update this!